api/utils/uaaClient.js

```diff
@@ -0,0 +1,51 @@
+const request = require('request');
+const url = require('url');
+const config = require('../../config');
+
+class UAAClient {
+  constructor(accessToken) {
+    this.accessToken = accessToken;
+  }
+
+  async verifyUserGroup(userId, groupNames = []) {
+    const { groups, origin, verified } = await this.request('GET', `/Users/${userId}`);
+    if (origin === 'cloud.gov' && !verified) {
+      return false;
+    }
+
+    return groups.filter(group => groupNames.indexOf(group.display) > -1).length > 0;
+  }
+
+  request(method, path, json) {
+    const host = config.env.uaaHost === 'http://localhost:9000'
```

Comment:
Want to find a cleaner way of setting this for the dev with docker network.

Reply:
hmmmm, i don't understand the issue here, isn't this request always made from the server? It's the redirect uri that needs to say localhost. What am I missing?

Reply:
Currently, the UAA host env creates the redirect path for `authorizationURL` for the browser to navigate to and it needs to be the localhost while the server does the rest of the calls to the docker network. Looking at it with fresh eyes, I'll just update the `authorizationURL` to be local host via the local config and we should be good.